src/core/path.c

```diff
@@ -289,3 +289,25 @@ QuicPathSetActive(
     CXPLAT_DBG_ASSERT(Path->DestCid != NULL);
     CXPLAT_DBG_ASSERT(!Path->DestCid->CID.Retired);
 }
+
+#ifdef QUIC_ROUTE_EXTRAS
+_IRQL_requires_max_(PASSIVE_LEVEL)
+QUIC_STATUS
+QuicPathResolveRoute(
```

Comment:
The function name is resolve route but it does both route lookup and next hop resolution.